Jenkinsfile

```diff
@@ -224,15 +242,15 @@ spec:
                             // $AZURE_SUBSCRIPTION_ID, $AZURE_TENANT_ID.
                             withCredentials([azureServicePrincipal('jenkins-bkpr-owner-sp')]) {
                                 def resourceGroup = 'jenkins-bkpr-rg'
-                                def clusterName = ("${env.BRANCH_NAME}".take(8) + "-${env.BUILD_NUMBER}-${platform}-" + UUID.randomUUID().toString().take(5)).replaceAll(/[^a-zA-Z0-9-]/, '-').toLowerCase()
+                                def clusterName = ("${env.BRANCH_NAME}".take(8) + "-${env.BUILD_NUMBER}-" + UUID.randomUUID().toString().take(5) + "-${platform}").replaceAll(/[^a-zA-Z0-9-]/, '-').toLowerCase()
```

Comment:
We need more uniqueness than platform+BUILD_NUMBER?  This expression is getting a bit silly - I think if we're going to include a random component, then lets just drop all the non-random bits and rely on tags/labels to report meta-info.

Reply:
The truncation removes the uniqueness. This change essentially is to make sure the uniqueness does not get removed as a result of truncations